### PR TITLE
Devise view

### PR DESCRIPTION
**Pull request by D-PALISSE**
finished login page, forgot password page and sign up page

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
-                autofocus: true,
-                input_html: { autocomplete: "email" } %>
-    <%= f.input :password,
-                required: false,
-                input_html: { autocomplete: "current-password" } %>
-    <%= f.input :remember_me, as: :boolean if devise_mapping.rememberable? %>
+      <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+      <div class="form-inputs">
+        <%= f.input :email,
+        required: false,
```
Comment (nassimbka) [inline app/views/devise/sessions/new.html.erb]: `required: true` non?

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+        autofocus: true,
+        input_html: { autocomplete: "email" } %>
+        <%= f.input :password,
+        required: false,
```
Comment (nassimbka) [inline app/views/devise/sessions/new.html.erb]: `required: true` here as well, no?

**Review by nassimbka — CHANGES_REQUESTED** [review]
Two comments on potentially required fields, otherwise looks OK to me (didn't review the front part, I trust your skills unless u need specific feedback (y) )

**Comment by D-PALISSE** [comment]
@nassimbka done, lets go merge if it's ok for you :+1:

**Comment by nassimbka** [comment]
All good, roger that. Fire in the hole !

**Pull request merged**